frontend/src/app/settings/page.tsx

```
@@ -618,106 +939,97 @@ export default function SettingsPage() {
-                  </p>
                 </div>
-                <motion.div
-                  className="relative inline-block w-12 h-6"
-                  whileHover={{ scale: 1.05 }}
-                  whileTap={{ scale: 0.95 }}
+
+                {/* Manage Subscription Button */}
+                <Link
+                  href="/manage-subscription"
+                  className="w-full px-6 py-4 bg-gradient-to-r from-purple-600 via-violet-600 to-purple-700 hover:from-purple-700 hover:via-violet-700 hover:to-purple-800 text-white rounded-xl font-semibold transition-all shadow-lg hover:shadow-xl flex items-center justify-center gap-3 group"
+                  role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)